projects/storefrontlib/src/lib/user/reset-password/reset-password-form/reset-password-form.component.html

```diff
@@ -0,0 +1,72 @@
+<form
+  (submit)="resetPassword()"
+  [formGroup]="form"
+  class="cx-reset-password-form-component"
+>
+  Enter a new password for account: email@address.com
```

Comment:
`Enter a new password for account: email@address.com`
This label does not make sense.  Let's either remove it or change it to something like `Enter a new password`

I guess the important part is to remove the ugly email placeholder because we won't have the user email on hand.